### PR TITLE
Deleted Tf and Cfn files, updated references to new Tf repo

### PR DESCRIPTION
**Pull request by eemperor**
Tf/Cfn files and folders removed and documentation references were updated to new [Watchmaker Terraform repository](https://github.com/plus3it/terraform-aws-watchmaker).

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -174,52 +174,64 @@ maps", and provide one for each of the templates above.
...
-Variable values can be input interactively via the terraform console or
-via a Terraform module. An example Terraform module that calls the
-lx-autoscale template is shown below.
+Variables can be input interactively via the Terraform console or
+directly to the Terraform module. An example Terraform file that calls the
+lx-autoscale module is shown below.
 
 ```
+provider "aws" {
+  profile = "default"
```
Comment (lorengordon) [inline docs/usage.md]: Can remove the "profile" line

File: docs/usage.md [hunk 2]
```diff
@@ -174,52 +174,64 @@ maps", and provide one for each of the templates above.
...
+
+The modules incorporate the CloudFormation templates within their respective Terraform templates so
+they become deployable and manageable from within the Terraform cli.
 
 .. note::
 
-   * These templates assume that the accompanying CloudFormation template
-     is in the same directory.
+   * Each corresponding Terraform template and the CloudFormation template are grouped together
+     in the same directory .
```
Comment (lorengordon) [inline docs/usage.md]: got an extra space before the period

Comment (eemperor) [reply docs/usage.md]: Space removed.

File: docs/usage.md [hunk 3]
```diff
@@ -214,9 +214,7 @@ directly to the Terraform module. An example Terraform file that calls the
 lx-autoscale module is shown below.
 
 ```
-provider "aws" {
-  profile = "default"
-}
+provide "aws" {}
```
Comment (lorengordon) [inline docs/usage.md]: => provide`r`

Comment (eemperor) [reply docs/usage.md]: Fixed!

**Pull request merged**